### PR TITLE
[feature] Add Tween.pause/resume/pauseAllByTarget/resumeAllByTarget method.

### PR DESCRIPTION
**Pull request by dumganhar**
Re: #17074

### Changelog

*

-------

### Continuous Integration

This pull request:

* [X] needs automatic test cases check.
  > Manual trigger with `@cocos-robot run test cases` afterward.
* [ ] does not change any runtime related code or build configuration
  > If any reviewer thinks the CI checks are needed, please uncheck this option, then close and reopen the issue.

-------

### Compatibility Check

This pull request:

* [ ] changes public API, and have ensured backward compatibility with [deprecated features](https://github.com/cocos/cocos-engine/blob/v3.5.0/docs/contribution/deprecated-features.md).
* [ ] affects platform compatibility, e.g. system version, browser version, platform sdk version, platform toolchain, language version, hardware compatibility etc.
* [ ] affects file structure of the build package or build configuration which requires user project upgrade.
* [ ] **introduces breaking changes**, please list all changes, affected features and the scope of violation.

### CODE REVIEW

**Comment by dumganhar** [comment]
I will add unit test for pauseAllByTarget/resumeAllByTarget and add API documentation for new API soon.

File: cocos/tween/actions/action.ts [hunk 1]
```diff
@@ -215,6 +213,10 @@ export abstract class Action {
         return this._id;
     }
 
+    setPaused (paused: boolean): void {
+        this._paused = paused;
+    }
```
Comment (minggo) [inline cocos/tween/actions/action.ts:218]: Why not use setter?

Comment (dumganhar) [reply cocos/tween/actions/action.ts:218]: Just keep the same API style in `Action` class.

Comment (minggo) [reply cocos/tween/actions/action.ts:218]: Ok.

**Review by minggo — APPROVED** [review]

**Comment by dumganhar** [comment]
@cocos-robot run test cases

**Pull request merged**